Release Notes/2017/2017-11-17-cloud-platform-release-notes.md

```diff
@@ -0,0 +1,96 @@
+
+  CenturyLink is happy to have successfully completed this significant upgrade to the GB1 node.  Customer instances in GB1 are now running in Reading on our latest-generation hyper-converged, hardware platform.  Our approach to capacity expansions is to make them completely in the background, as is expected of a cloud service.  Extensive network engineering focus and planning went into ensuring all resources were auto-balanced into the upgraded infrastructure seamlessly, without downtime or impact to customer instances.  Since the upgrade was completed, we have seen significant performance improvements and have confirmed this with benchmarking by Cloud Spectator, who showed a significant performance of our GB1 node compared to the prior assessment conducted in May 2017 on the older node architecture.
+
+  We recognize that this capacity expansion, while beneficial to customer application performance, has changed the physical location of GB1-hosted workloads.  If this alteration to the geography of the CenturyLink Cloud platform introduces any issues around node deployment patterns, we would be happy to work with customers on alternatives which leverage other parts of our global footprint.
+
+* __Removed 3PAR QOS 4 for Utility Storage__
+
+  Customers will no longer be able to order 3PAR QOS 4 when ordering Utility Storage when using the DCC UI or SavvisStation Portal.
+
+* __Notice of Microsoft SQL Server Fee Change__
+
+  *The following notice was sent to all MSSQL clients on Thursday, November 16th.*  
+
+  Beginning [60 days following notice date], 2018, the service fees for Microsoft SQL Server on CenturyLink Cloud will change. As a result of a recent rate increase notification from Microsoft, we have re-evaluated our pricing for Microsoft SQL Server and are implementing the following price change to account for the increase and adjust based on the overall market .
+  Customers will be billed the following new rate for any existing and new Microsoft SQL Server deployments.
+
+  ![](../../images/release_notes/2017-11-17/MSSQL_rate_change.png)
+
+  Should you have additional questions, please contact your Sales Representative or email help@ctl.io.
+
+__Cloud Application Manager__
+
+* __AWS Optimization and Analytics available for U.K. based customers__
+
+  AWS Optimization and Analytics for companies legally based in the United Kingdom is Live! From the AWS Provider, CAM org administrators, will be permitted to automatically complete [share-shift migrations](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-aws-existing/) or provision [new]( https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-aws-new/) AWS accounts for consolidated billing, analytics, and platform level support. 
```

Comment:
Change links to `(../../Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-existing.md)` and `(../../Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-new.md)` respectively.